meta-resin-common/recipes-containers/balena/balena_git.bb

```diff
@@ -10,7 +10,7 @@ pulling."
 inherit binary-compress
 FILES_COMPRESS = "/boot/init"
 
-SRCREV = "9c78e9b21021d2d2bbd17ebacc7ba9eacacc0742"
+SRCREV = "ac3c27b1bed3222a5cf9fc18fe6f65d7b2a6d1d8"
```

Comment:
This commit can be merged with the 1st commit: https://github.com/resin-os/meta-resin/pull/902/commits/7eea605789c74a371ec9b472473c21513d6ab862 (balena: Replace docker with balena)

Reply:
Now that I see the next commit is undoing this commit here, why not just remove both commits from this PR?

Reply:
No, I don't plan to squash this PR.